packages/video_player/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayerPlugin.java

```diff
@@ -238,11 +238,15 @@ public void onMethodCall(MethodCall call, Result result) {
           VideoPlayer player;
           if (call.argument("asset") != null) {
             try {
-              AssetFileDescriptor fd =
-                  registrar
-                      .context()
-                      .getAssets()
-                      .openFd(registrar.lookupKeyForAsset((String) call.argument("asset")));
+              String assetLookupKey;
+              if (call.argument("package") != null) {
+                assetLookupKey =
+                    registrar.lookupKeyForAsset(
```

Comment:
Probably not applicable, just for future reference:

what does `registrar.lookupKeyForAsset("assetname", null)` do? Could you do just always: 
```
assetLookupKey =registrar.lookupKeyForAsset((String) call.argument("asset"),  (String) call.argument("package"));
```